virtool/http/client.py

```diff
@@ -137,23 +152,27 @@ async def has_right_on_upload(self, upload_id: str, right: Right) -> bool:
 
 class JobClient(AbstractClient):
 
-    def __init__(self, ip: str, job_id, rights: JobRights):
-        self._ip = ip
+    def __init__(self, job_id, rights: JobRights):
         self._rights = rights
 
         self.job_id = job_id
+        self.permissions = dict()
+
+    @property
+    def authenticated(self) -> bool:
+        return True
 
     @property
     def administrator(self):
         return False
 
     @property
-    def ip(self):
-        return self._ip
-
-    def has_permission(self, permission) -> bool:
+    def force_reset(self) -> bool:
         return False
 
+    def has_permission(self, permission: str) -> bool:
+        return self.permissions.get(permission, False)
```

Comment:
This should just return `False`. Jobs don't have permissions, only rights on specific resources. Right?